pkg/engine/context/imageutils.go

```diff
@@ -186,8 +189,12 @@ func newImageInfo(image, jsonPointer string) (*ImageInfo, error) {
 		digest = digested.Digest().String()
 	}
 
+	if digest != "" && tag == "" {
```

Comment:
This expression is essentially a no-op. If the tag is already an empty string we are not making any changes to it by reassigning it to an empty string.

Reply:
Understood, will surely fix it!